oedisi/tools/cli_tools.py

```diff
@@ -126,7 +320,43 @@ def run_with_pause(runner):
     broker.run()
     background_runner.wait()
 
-
+@cli.command()
+@click.option("--runner", 
+    default="build/docker-compose.yml", # build/kubernetes/deployment.yml
+    type=click.Path(),
+    help="path to the docker-compose or kubernetes deployment file"
+)
+@click.option(
+    "-k", "--kubernetes",
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="Use the flag to launch in a kubernetes pod. "
+)
+@click.option(
+    "-d", "--docker-compose",
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="Use the flag to launch in a kubernetes pod. "
+)
+def run_mc(runner, kubernetes, docker_compose):
```

Comment:
We probably shouldn't keep this around since it's a bit dangerous: it calls docker system prune --all for instance.

Reply:
We'll remove this later in the next refactor.